helm/hpcc/templates/dafilesrv.yaml

```diff
@@ -89,5 +89,12 @@ kind: ConfigMap
 {{ include "hpcc.addService" ( dict "root" $ "name" .name "service" $service "selector" .name "defaultPort" 7600 ) }}
 ---
 {{ include "hpcc.addCertificate" (dict "root" $ "name" .name "service" .service "component" "dafilesrv" "visibility" .service.visibility) }}
+
+{{- $instance := .name -}}
+{{- $visibility := .service.visibility -}}
+{{- range $remoteClient := .remoteClients }}
+ {{ include "hpcc.addClientCertificate" (dict "root" $ "client" $remoteClient.name "instance" $instance "component" "dafilesrv" "visibility" $visibility) }}
```

Comment:
formatting: looks like the helm indentation may be wrong here, would expect perhaps it to be {{- include ..... -}}